src/Controls/samples/Controls.Sample.Droid/Maui.Controls.Sample.Droid-net6.csproj

```diff
@@ -1,11 +1,12 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.Razor">
 
   <PropertyGroup>
     <TargetFramework>net6.0-android</TargetFramework>
     <OutputType>Exe</OutputType>
     <RootNamespace>Maui.Controls.Sample.Droid</RootNamespace>
     <AssemblyName>Maui.Controls.Sample.Droid</AssemblyName>
     <IsPackable>false</IsPackable>
+    <StaticWebAssetsEnabled>true</StaticWebAssetsEnabled>
```

Comment:
Hmm, not sure I understand why this is needed, it should be enabled based on the fact you are referencing the SDK